tailcfg/tailcfg.go

```diff
@@ -624,11 +624,12 @@ func (h *Hostinfo) CheckRequestTags() error {
 type ServiceProto string
 
 const (
-	TCP          = ServiceProto("tcp")
-	UDP          = ServiceProto("udp")
-	PeerAPI4     = ServiceProto("peerapi4")
-	PeerAPI6     = ServiceProto("peerapi6")
-	PeerAPIDNS   = ServiceProto("peerapi-dns-proxy")
+	TCP        = ServiceProto("tcp")
+	UDP        = ServiceProto("udp")
+	PeerAPI4   = ServiceProto("peerapi4")
+	PeerAPI6   = ServiceProto("peerapi6")
+	PeerAPIDNS = ServiceProto("peerapi-dns-proxy")
+	// Deprecated: use the field on HostInfo instead.
```

Comment:
toss a date/capver/issue URL on this? was this ever in a released client? if so, something like "used until 1.52.x".

Reply:
It hasn't been released yet, and we can avoid retaining it. Once we've rolled this through control I'll come back and remove it. Currently connectors are only supported in unstable and we can put a capver gate on them when we change that